src/main/java/com/playmonumenta/libraryofsouls/bestiary/BestiaryCommand.java

```diff
@@ -16,6 +16,7 @@
 import java.util.ArrayList;
 import java.util.List;
 import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.TextComponent;
```

Comment:
I don't know why the review dog isn't properly reporting these anymore - this unused import needs to be removed.